### PR TITLE
BugFix: Seeds run

### PR DESCRIPTION
**Pull request by chrysalag**
# [Story in Trello](https://trello.com/c/RCPZiDl6/49-seed-errors)

### Done for approval?

* [x] Acceptance criteria are met
* [x] Code quality checks are green
* [x] Readme updated if needed
* [ ] Story under test
* ~[ ] Mobile works~
* [x] Seeds created
* ~[ ] Translated~
* [ ] Code is reviewed

### Done after the merge?

* [ ] Deployed to staging after the merge
* [ ] Tested manually on staging

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -43,10 +43,8 @@ def make_lang_skills
 end
 
 [:superadmin, :social_worker, :department_manager].each do |role|
-  user = FactoryBot.create(:user, role: role, email: "#{role}#{EMAIL_DOMAIN}",
+  FactoryBot.create(:user, role: role, email: "#{role}#{EMAIL_DOMAIN}",
     password: 'asdfasdf')
-  user.profile.contact.primary_email = "#{role}#{EMAIL_DOMAIN}"
```
Comment (casaper) [inline db/seeds.rb]: The Primary email should be set, in order to have views propperly displayed...

Comment (chrysalag) [reply db/seeds.rb]: `primary_email` is set. I checked in the console for users that where created through the seeds and their `primary_email` was set. I could login and change the primary email for superadmin, department_manager, social_worker and volunteer. Could you please provide some info on how I could reproduce the issues you mention?

Comment (casaper) [reply db/seeds.rb]: is it the same email as User.email? I assume not

Comment (chrysalag) [reply db/seeds.rb]: `User.email` and `User.profile.contact.primary_email`are the same for `superadmin`, `department_manager` and `social_worker` users.

Comment (casaper) [reply db/seeds.rb]: I found out, that its the callback... I was assuming the factory to hit in a random email, and forgot about that callback that updates the email.
Sorry about that!

**Pull request merged**